axfr.go

```diff
@@ -14,6 +14,34 @@ func (g *Gandi) ListTsigs() (tsigs []Tsig, err error) {
 	return
 }
 
+// By gandi API design, an axfr/tsig/uuid/config/[bind|knot|powerdns|nsd] request will not return a valid JSON,
```

Comment:
I don't think this comment is useful if you use `askGandiToBytes`. It would be more or less self-documented.